### PR TITLE
OSDOCS-4397:Add ZRS support for Azure Disk CSI

### PR DESCRIPTION
**Pull request by lpettyjo**
Version(s): 4.10+

Issue: https://issues.redhat.com/browse/OSDOCS-4397

Link to docs preview: https://57216--docspreview.netlify.app/openshift-enterprise/latest/storage/container_storage_interface/persistent-storage-csi-azure.html#persistent-storage-csi-azure-disk-sc-zrs_persistent-storage-csi-azure

QE review:
- [ ] QE has approved this change.

Additional information:

**PTAL**:  @duanwei33, @jsafrane, @gcharot, @sferich888, @mwerner2113

### CODE REVIEW

**Comment by gcharot** [comment]
LGTM

File: modules/persistent-storage-csi-azure-disk-sc-zrs.adoc [hunk 1]
```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
```
Comment (skrthomas) [inline modules/persistent-storage-csi-azure-disk-sc-zrs.adoc:1]: You need a :_content-type: PROCEDURE here

**Review by skrthomas — COMMENTED** [review]
One small thing, LGTM otherwise. Nice work :)

**Comment by mwerner2113** [comment]
lgtm

**Comment by jsafrane** [comment]
lgtm

File: modules/persistent-storage-csi-azure-disk-sc-zrs.adoc [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+//
+
+:_content-type: PROCEDURE
+[id="persistent-storage-csi-azure-disk-sc-zrs_{context}"]
+= Creating a storage class with storage account type
+
+
+Storage classes are used to differentiate and delineate storage levels and usages. By defining a storage class, you can obtain dynamically provisioned persistent volumes.
+
+When creating a storage class, you can designate the storage account type. This corresponds to your Azure storage account SKU tier. Valid options are `Standard_LRS`, `Premium_LRS`, `StandardSSD_LRS`, `UltraSSD_LRS`, `Premium_ZRS`, `StandardSSD_ZRS`, and `PremiumV2_LRS`.
```
Comment (sferich888) [inline modules/persistent-storage-csi-azure-disk-sc-zrs.adoc:14]: How does one check there 'Azure storage account SKU tier'?

**Pull request merged**